src/framework/mlt_repository.h

```diff
@@ -50,7 +50,7 @@ typedef mlt_properties (*mlt_metadata_callback)(mlt_service_type,
                                                 void * /* callback_data */);
 
 /** A convenience macro to create an entry point for service registration. */
-#define MLT_REPOSITORY void mlt_register(mlt_repository repository)
+#define MLT_REPOSITORY MLT_API void mlt_register(mlt_repository repository)
```

Comment:
To fix the problem Dan faced I believe we need to revert the change in this line and add export headers for every module and use them like

`MLTFREI0R_EXPORT MLT_REPOSITORY`